### PR TITLE
Arcsjs Meet Plugin

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: pkg/extensions/meet/arcs/ExtApp.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  HistoryService,
+  ThreejsService,
+  ShaderService,
+  TensorFlowService,
+  SelfieSegmentationService,
+  MediaService
+} from './arcs.js';
+import {ExtRecipe} from './ExtRecipe.js';
+
+export const ExtApp = class extends App {
```
Comment (mariakleiner) [inline pkg/extensions/meet/arcs/ExtApp.js:18]: wondering - this is the generic Extension app, right? or else what is meet specific here (except recipe, of course) ?

**Review by mariakleiner — APPROVED** [review]
\🤯 /

File: pkg/extensions/meet/arcs/ExtApp.js (hunk 1 above)
Comment (sjmiles) [reply pkg/extensions/meet/arcs/ExtApp.js:18]: At some level App is just a vehicle for configuration: the recipe, the services, the persistor, the Path maps.
This is an App that pulls those things together for the Meet experience I made.

There is also this, but it's not used right now:
`this.arcs.set('user', 'mediaDeviceState', {isCameraEnabled: true});`

Comment (sjmiles) [reply pkg/extensions/meet/arcs/ExtApp.js:18]: Is actually just a generic app. It has an `index.html` so you can run it without extension wrapper (but it doesn't do a whole lot unless you supply inputs).

**Pull request merged**